keras_cv/layers/preprocessing/channel_shuffle_test.py

```diff
@@ -0,0 +1,88 @@
+import tensorflow as tf
+from keras_cv.layers.preprocessing.channel_shuffle import ChannelShuffle
+
+
+class ChannelShuffleTest(tf.test.TestCase):
```

Comment:
Let's add a test case to test it in a `@tf.function` or a `keras.Model`

Reply:
There is a function inside this class called `test_in_tf_function`, is that what you're asking?